discovery-provider/src/tasks/index_network_peers.py

```diff
@@ -24,10 +24,14 @@
 
 # Perform eth web3 call to fetch endpoint info
 def fetch_cnode_info(sp_id, sp_factory_instance):
+    redis = update_network_peers.redis
     cn_endpoint_info = sp_factory_instance.functions.getServiceEndpointInfo(
         content_node_service_type,
         sp_id
     ).call()
+    sp_id_key = get_sp_id_key(sp_id)
+    pickle_and_set(redis, sp_id_key, cn_endpoint_info, 50000)
```

Comment:
50k is like 13 hours. i can see that being okay, maybe a little high, but was this intentional?

Reply:
also this is literally overriding the value in redis every 30 seconds. was this also intentional? one suggestion is if we made this TTL like 30 minutes or something, we could call `get_pickled_key(sp_id_key)` and only if the key doesn't exist fetch from SPFactory and write to redis. that would significantly reduce our web3 overhead as well

Reply:
sounds good